examples/speaker_recognition/speaker_reco.py

```diff
@@ -43,15 +44,20 @@
 
 """
 
+seed_everything(42)
+
 
 @hydra_runner(config_path="conf", config_name="config")
 def main(cfg):
 
     logging.info(f'Hydra config: {cfg.pretty()}')
     trainer = pl.Trainer(**cfg.trainer)
-    exp_manager(trainer, cfg.get("exp_manager", None))
+    log_dir = exp_manager(trainer, cfg.get("exp_manager", None))
     speaker_model = EncDecSpeakerLabelModel(cfg=cfg.model, trainer=trainer)
     trainer.fit(speaker_model)
+    model_path = os.path.join(log_dir, '..', 'spkr.nemo')
+    speaker_model.save_to(model_path)
+    trainer.test(speaker_model)
```

Comment:
May want to run test on a new Trainer with a single GPU to get correct scores + no effects of DDP. Look at how speech recognition example handles this.

Reply:
why there are effects of DDP on test but not on valid?